test/features/pagecontent/save_api.js

```diff
@@ -251,10 +272,28 @@ describe('page save api', function() {
         });
     });
 
+    it('should forbid external usage', function() {
```

Comment:
> So, the idea is that RESTBase should allow page saves for internal clients and fail for others.

No, the POST end point restriction is not related to XFF forwarding. They are two independent issues that just ended up in the same PR.